release/Makefile

```diff
@@ -0,0 +1,107 @@
+release_builds := \
+	dist/operator-sdk-$(GIT_VERSION)-aarch64-linux-gnu \
+	dist/operator-sdk-$(GIT_VERSION)-x86_64-linux-gnu \
+	dist/operator-sdk-$(GIT_VERSION)-x86_64-apple-darwin \
+	dist/operator-sdk-$(GIT_VERSION)-ppc64le-linux-gnu \
+	dist/operator-sdk-$(GIT_VERSION)-s390x-linux-gnu \
+	dist/ansible-operator-$(GIT_VERSION)-aarch64-linux-gnu \
+	dist/ansible-operator-$(GIT_VERSION)-x86_64-linux-gnu \
+	dist/ansible-operator-$(GIT_VERSION)-x86_64-apple-darwin \
+	dist/ansible-operator-$(GIT_VERSION)-ppc64le-linux-gnu \
+	dist/ansible-operator-$(GIT_VERSION)-s390x-linux-gnu \
+	dist/helm-operator-$(GIT_VERSION)-aarch64-linux-gnu \
+	dist/helm-operator-$(GIT_VERSION)-x86_64-linux-gnu \
+	dist/helm-operator-$(GIT_VERSION)-x86_64-apple-darwin \
+	dist/helm-operator-$(GIT_VERSION)-ppc64le-linux-gnu \
+	dist/helm-operator-$(GIT_VERSION)-s390x-linux-gnu
+
+.PHONY: release
+release: $(release_builds) $(release_builds:=.asc) ## Release the Operator SDK
+
+dist/operator-sdk-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
+dist/operator-sdk-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
+dist/operator-sdk-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
+dist/operator-sdk-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
+dist/operator-sdk-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
+dist/operator-sdk-%-linux-gnu: GOARGS = GOOS=linux
+
+dist/ansible-operator-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
+dist/ansible-operator-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
+dist/ansible-operator-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
+dist/ansible-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
+dist/ansible-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
+dist/ansible-operator-%-linux-gnu: GOARGS = GOOS=linux
+
+dist/helm-operator-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
+dist/helm-operator-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
+dist/helm-operator-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
+dist/helm-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
+dist/helm-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
+dist/helm-operator-%-linux-gnu: GOARGS = GOOS=linux
+
+dist/%: ## Build the operator-sdk release binaries
+	{ \
+	cmdpkg=$$(echo $* | sed -E "s/(operator-sdk|ansible-operator|helm-operator).*/\1/"); \
+	go build $(GO_BUILD_ARGS) -o $@ ./cmd/$$cmdpkg; \
+	}
+
+dist/%.asc: ## Create release signatures for operator-sdk release binaries
+	{ \
+	default_key=$$(gpgconf --list-options gpg | awk -F: '$$1 == "default-key" { gsub(/"/,""); print toupper($$10)}'); \
+	git_key=$$(git config --get user.signingkey | awk '{ print toupper($$0) }'); \
+	if [ "$${default_key}" = "$${git_key}" ]; then \
+		gpg --output $@ --detach-sig dist/$*; \
+		gpg --verify $@ dist/$*; \
+	else \
+		echo "git and/or gpg are not configured to have default signing key $${default_key}"; \
+		exit 1; \
+	fi; \
+	}
```

Comment:
nit: would be helpful to have the help target in the makefile, but I suspect we shouldn't be interacting with this file directly. I did find myself going into `release` and running `make help` to see what was in it :)

```
.PHONY: help
help: ## Show this help screen.
    @echo 'Usage: make <OPTIONS> ... <TARGETS>'
    @echo ''
    @echo 'Available targets are:'
    @echo ''
    @awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Reply:
I'll add this in #4034 since I'd like that Makefile to be called directly.

Reply:
+1 for the suggestion